parent/main/resources/checkstyle.xml

```diff
@@ -36,13 +36,6 @@
       <!-- Checks for imports                              -->
       <!-- See http://checkstyle.sf.net/config_import.html -->
       <module name="IllegalImport"/> <!-- defaults to sun.* packages -->
-      <module name="ImportOrder">
-         <property name="groups" value="java,javax,*"/>
-         <property name="ordered" value="true"/>
-         <property name="separated" value="false"/>
-         <property name="option" value="bottom"/>
-         <property name="sortStaticImportsAlphabetically" value="true"/>
-      </module>
```

Comment:
@rmacor The ordering can be changed in Editor/Code style/Java tab Imports. To not mess with imports, go to Editor/General/Auto Import and uncheck "Optimize imports on the fly".